vignettes/frictionless.Rmd

```diff
@@ -256,24 +353,28 @@ Note that in the above steps you started a Data Package from scratch with `creat
 
 Now that you have created your Data Package, you can write it to a directory of your choice with `write_package()`:
 
-```{r}
+
+```r
 write_package(my_package, "my_directory")
+#> Error in `write_package()`:
+#> ! `package` must have resources.
+#> ℹ Use `add_resource()` to add resources.
 ```
 
 The directory will contain four files: the descriptor `datapackage.json`, one CSV file containing the data for the resource `iris` and two CSV files containing the data for the resource `observations`.
 
+
 ```r
 list.files("my_directory")
-#> [1] "datapackage.json"   "iris.csv"           "observations_1.csv"
-#> [4] "observations_2.csv"
+#> [1] "datapackage.json"   "iris.csv"           "observations_1.csv" "observations_2.csv"
```

Comment:
If I run `compile.R` I get `character(0)`.

Reply:
Caused by incorrect function call, fixed in d781c83.